src/librustdoc/config.rs

```diff
@@ -341,6 +341,14 @@ impl Options {
         // check for deprecated options
         check_deprecated_options(&matches, &diag);
 
+        // check for `--output-format=json`
+        if matches.opt_present("output-format") && !matches.opt_present("show-coverage") && !nightly_options::is_unstable_enabled(matches) {
```

Comment:
`--output-format=html` has been stable since 1.0.0

Reply:
Thanks! I'll change it just to check for `json`.